modules/ROOT/pages/maven-reference.adoc

```diff
@@ -0,0 +1,169 @@
+
+This page summarizes reference information that helps work with Maven and Mule. Refer to xref:using-maven-with-mule.adoc[Using Maven with Mule] for an introduction and overview.
+
+== Deploying Mule Artifacts to a Central Maven Repository
+
+Using a central repository can save users' time when building applications. To deploy to a central repository, you need to deploy the Mule `POM` files and `JAR` artifacts to the desired repository.
+
+The deploy plugin is primarily used during the deploy phase, to add your artifacts to a remote repository for sharing with other developers and projects.
+
+=== Deploying pom.xml
+
+To deploy the `pom.xml` to a remote repository, use the following command:
+
+[source,text,linenums]
+----
+mvn deploy:deploy-file
+-DgroupId=${project.groupId}
+-DartifactId=${project.artifactId}
+-Dversion=${version}
+-Dpackaging=pom
+-Dfile=${localPom.canonicalPath}
+-Durl=http://<your_repository_ip>/nexus/content/repositories/releases
+-DrepositoryId=releases
+----
+
+For the `-Durl` parameter, replace `<your_repository_ip>` with the correct IP address. Also, the `-DrepositoryId` needs to match with the credentials defined on the `settings.xml` file.
+
+=== Deploying the JAR files
+
+To deploy a `JAR` file to a remote repository, use the following command:
+
+[source,text,linenums]
+----
+mvn deploy:deploy-file
+-DgroupId=${pomProps.groupId}
+-DartifactId=${pomProps.artifactId}
+-Dversion=${pomProps.version}
+-Dpackaging=jar
+-Dfile=${f.canonicalPath}
+-DpomFile=${localPom.canonicalPath}
+-Durl=http://${your_repository_ip}/nexus/content/repositories/releases
+-DrepositoryId=releases
+----
+
+For the `-Durl` parameter, replace `your_repository_ip` with the correct IP address. Also, the `-DrepositoryId` needs to match with the credentials defined on the `settings.xml` file.
+
+If you wish to both install the `JAR` files locally and deploy them remotely, you can keep both locations in the argument definition.
+
+== pom.xml File
+
+[source,xml]
+----
+<project root>/pom.xml
+----
+
+Project Object Model file that defines settings for a Maven project describing an application. It includes all settings necessary to build the application such as build plugin configurations. Note that the `pom.xml` exists on a per-project basis and is distributed along with a project.
+
+== settings.xml File
+
+[source,xml,linenums]
+----
+<system-wide Maven directory>/settings.xml
+<user home directory>/.m2/settings.xml
+----
+
+Contains global settings for your Maven installation. Unlike a project's `pom.xml`, it defines system-wide settings and is not distributed with a project, since it can contain confidential information such as authentication credentials.
+
+The `settings.xml` file may reside in two locations:
+
+* In a system-wide settings folder:
++
+In this case, it is a _global_ settings file which defines the settings for all Maven installations on the system, regardless of user. For example:
```

Comment:
... a global settings file **that** defines...
Should we tag the file as `settings.xml`?